implementations/code_df/issue_response_time_github.py

```diff
@@ -0,0 +1,114 @@
+from datetime import datetime
+import numpy as np
+
+import utils
+from issue import Issue
+
+
+class IssueResponseTimeGithub(Issue):
+    """
+    Class for Issue Response Time
+    """
+
+    def _flatten(self, item):
+        """
+        Flatten a raw issue fetched by Perceval into a flat dictionary.
+
+        A list with a single flat directory will be returned.
+        That dictionary will have the elements we need for computing metrics.
+        The list may be empty, if for some reason the issue should not
+        be considered.
+
+        :param item: raw item fetched by Perceval (dictionary)
+        :returns:   list of a single flat dictionary
+        """
+
+        creation_date = utils.str_to_date(item['data']['created_at'])
+        if self.since and (self.since > creation_date):
+            return []
+
+        if self.until and (self.until < creation_date):
+            return []
+
+        # all pull requests are considered to be issues by default.
+        # this check removes pull requests from the
+        # analysis.
+        if 'pull_request' in item['data']:
+            return []
```

Comment:
Here I am removing pull requests from the issues data (since GitHub treats all pull requests as issues too). Hope this is fine. (this check will of course be in the `Issue_Github` category class (in `_flatten`) and is not specific to this metric.)

Reply:
As in other cases, i think it is easier for this reference implementation to just assume that the class is fed only with prs. It is easy to do that before instantiating the class. So, please remove that check...